03_probability_intro/03_probability_intro.jl

```diff
@@ -171,22 +384,63 @@ This may sound a bit confusing at first, but if you look closely for some time a
 What is nice about the Bayesian framework is that we always account for the uncertainty of the world. We start with some probability and end with another probability, but uncertainty is always present. This is what real life is about.
 To understand the full power of Bayesian probability we have to extend the notion of probability to *probability density*. We will discuss this topic in the following section.
 "
 
 # ╔═╡ 2d9482ce-1252-11eb-0cc7-35ad9c288ef8
 md"
-# Probability density
+# Probability distributions
 So far, we have been talking of probabilities of particular events.
-Probability density, on the other hand, help us compute probabilities of various events.
-These are functions that connect each event in an 'event space' to some probability. What do we mean by 'event space'? For example, consider the density probability of heights of adult women, given approximately by a **Normal distribution**,
+Probability distributions, on the other hand, help us compute probabilities of various events.
+We can distinguish between discrete and continuous cases depending on the possible output of the experiment.
 "
 
-# ╔═╡ 351923ee-5436-11eb-2bf6-8d024a64e83e
+# ╔═╡ f36ea8bc-9623-11eb-24a2-155b112db75b
+md"
+## Discrete Case
+If the outputs of our experiment are discrete, then its distribution is called a probability mass function, where we assign a probability to each possible outcome.
+
+One of the most popular distributions is the Poisson distribution.
+Suppose I want to visualize the probability of receiving *x* spam mails on Mondays.
+"
+
+
+# ╔═╡ e369ca40-9624-11eb-3d71-7719b56e0559
 md"
+Here we represent the probability of receiving *x* spam mail in a day.
+The interpretation of this graph is pretty straightforward. 
+The probability of receiving 0 spam emails on a Monday is approximately 0.2, for 1 spam email is slightly higher than 0.3 and so on, we have the probability of each possible output.
+
+By replicating the code below in a Pluto notebook, you will be able to create sliders to play around with the values of μ and σ and see how the Poisson distribution changes.
+
+"
+
+
+
+# ╔═╡ 59b04ef6-962a-11eb-29ce-8d401d3a59d7
+@bind  λ	html"<input type=range min=0.5 max=5 step=0.1>"
+
+# ╔═╡ 0aa945c6-962b-11eb-3313-3deb23234ac5
+bar(Poisson(λ), xlabel="x", ylabel="Probability", legend=false, size=(400, 300))
 
+# ╔═╡ e356a712-9624-11eb-15a9-49a7548e0b5b
+md"## Continuous cases
+Instead of a probability mass function, a continuous random variable has a probability density function.
```

Comment:
the concept of random variable was never introduced. It must be introduced first somewhere or this line should changed

Reply:
It is also important to talk about that it is pointless to talk about probability of a single x value (in a continuous case). Probability in the continuous case is always computed in an interval, at leaste in an infinitesimal one (from x to x + dx, for example)

Reply:
When we work with continuous variables it is pointless to talk about the probability of a single x value. 
Think of it in a mathematical way, in a number line there are infinity points in between 0 and 0,01. 
In this case, our continuous variable is women's height, since there are infinitely possible heights it has no sense to talk about the probability of a single height, like $P(6 in)$.

Probability in the continuous case is always computed in an interval.
For example, suppose we want to know the probability that a randomly selected woman measures between 60 and 65 inches.
To know it we need to calculate the area under the density curve in the intervals x = [60,65].